agent/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Instrumentation Agent for Java
+
+[![Build Status][travis-image]][travis-url] [![Maven Central][maven-image]][maven-url]
+
+The *Google Instrumentation Agent for Java* collects and sends latency data
+about your Java process to Stackdriver Trace for analysis and visualization
```

Comment:
This is now part of https://github.com/census-instrumentation/instrumentation-java/pull/364, using the OpenCensus branding.